### PR TITLE
[5.2] [Guided tours] Bug fixes for PR #43966

### PR DESCRIPTION
**Pull request by tecpromotion**
Pull Request for PR #43966

### Summary of Changes

Add missing file header
Correction of the spelling of the language strings
add "noreferrer" to link

### Testing Instructions

It should have no effect on the functionality.

### Actual result BEFORE applying this Pull Request

language_string = ""

### Expected result AFTER applying this Pull Request

language_string=""

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Review by richard67 — APPROVED** [review]

**Comment by brianteeman** [comment]
Nested Subforms - Allows the nesting of Subform inside a Subform Custom Field.
==>
Nested Subforms - Allows the nesting of a Subform inside a Subform Custom Field.

**Review by obuisard — APPROVED** [review]
Thank you for the improvements.

**Comment by obuisard** [comment]
> Nested Subforms - Allows the nesting of Subform inside a Subform Custom Field. ==> Nested Subforms - Allows the nesting of a Subform inside a Subform Custom Field. or Nested Subforms - Allows the nesting of Subforms inside a Subform Custom Field.
> 
> I think its the first option assuming you can only nest one subform otherwise its the second

The PR #43204 allows the selection of subform fields in a subforms. There could be more than one. So that should be plural.

File: administrator/language/en-GB/guidedtours.joomla_whatsnew_5_2.ini [hunk 1]
```diff
@@ -1,2 +1,7 @@
...
.../div><div class=\"row align-items-center border-top border-1 pt-4\"><div class=\"col-md-6 col-lg-7 mb-3\"><h2 class=\"mb-3\">Core Updates</h2><h3>The new all-in-one 'Articles' module</h3><p>Use one module to display latest, most read, specific or archived articles, from select categories - with filters, tags and more...</p><p>From <span class="fw-medium">Content -&gt; Site Modules</span>, create a new module of type 'Articles'.</p><h3>Mail Template Layout</h3><p>Take it a step further by creating an entirely unique layout (including logo) which can be applied to one or all mail templates.</p><p>From <span class=\"fw-medium\">Global Configuration -&gt; Mail Templates</span>, layouts can be defined in HTML.</p><h3>Other Enhancements and Fixes</h3><ul class=\"mb-3\"><li>Nested Subforms - Allows the nesting of Subform inside a Subform Custom Field.</li><li>Total Count - Shows the count and total number of list items in the console.</li></ul></div><div class=\"col-md-6 col-lg-5\"><img class=\"img-fluid\" src=\"media/com_guidedtours/images/5_2/mailtemplate.png\" alt=\"Mail template example, including a header logo and a footer\" width=\"640\" height=\"370\" loading=\"lazy\"></div></div>"
```
Comment (brianteeman) [inline administrator/language/en-GB/guidedtours.joomla_whatsnew_5_2.ini]:
```suggestion
COM_GUIDEDTOURS_TOUR_WHATSNEW_5_2_DESCRIPTION="<h1 class=\"fw-medium\">Welcome to Joomla 5.2!</h1><div class=\"row align-items-center mb-4 pt-4 border-top border-1\"><div class=\"col-md-6 col-lg-5 text-center text-md-start mb-3 mb-md-0\"><img class=\"img-fluid\" src=\"media/com_guidedtours/images/5_2/enhancedtourheader.png\" alt=\"The first step of the welcome tour, showing the new header look\" width=\"640\" height=\"384\"></div><div class=\"col-md-6 col-lg-7\"><h2 class=\"mb-3\">Guided Tours Updates</h2><h3>Auto-start Tours</h3><p>This feature will allow you to create your own auto-start Guided Tour. The contextual tour can be opted out for one or all users. Tours can be restarted from the Guided Tours menu.</p><h3>Enhancements</h3><ul class=\"mb-3\"><li>For clarity, the <em>Step Counter</em> was positioned first in the tour header.</li><li>The modal window resizes automatically to accommodate larger content and images.</li></ul></div></div><div class=\"row align-items-center border-top border-1 pt-4\"><div class=\"col-md-6 col-lg-7 mb-3\"><h2 class=\"mb-3\">Core Updates</h2><h3>The new all-in-one 'Articles' module</h3><p>Use one module to display latest, most read, specific or archived articles, from select categories - with filters, tags and more...</p><p>From <span class="fw-medium">Content -&gt; Site Modules</span>, create a new module of type 'Articles'.</p><h3>Mail Template Layout</h3><p>Take it a step further by creating an entirely unique layout (including logo) which can be applied to one or all mail templates.</p><p>From <span class=\"fw-medium\">Global Configuration -&gt; Mail Templates</span>, layouts can be defined in HTML.</p><h3>Other Enhancements and Fixes</h3><ul class=\"mb-3\"><li>Nested Subforms - Allows the nesting of Subforms inside a Subform Custom Field.</li><li>Total Count - Shows the count and total number of list items in the console.</li></ul></div><div class=\"col-md-6 col-lg-5\"><img class=\"img-fluid\" src=\"media/com_guidedtours/images/5_2/mailtemplate.png\" alt=\"Mail template example, including a header logo and a footer\" width=\"640\" height=\"370\" loading=\"lazy\"></div></div>"
```

File: administrator/language/en-GB/guidedtours.joomla_whatsnew_5_2_steps.ini [hunk 2]
```diff
@@ -1,2 +1,7 @@
...
-COM_GUIDEDTOURS_TOUR_WHATSNEW_5_2_STEP_0_TITLE = "More Information and Help"
+; Joomla! Project
+; (C) 2024 Open Source Matters, Inc. <https://www.joomla.org>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+COM_GUIDEDTOURS_TOUR_WHATSNEW_5_2_STEP_0_DESCRIPTION="<h3>Joomla 5.2 Full Release Notes</h3><p><a title=\"Click to view full release notes\" href=\"https://www.joomla.org/announcements.html\" target=\"_blank\" rel=\"noopener noreferrer\">View the full release notes on joomla.org</a></p><hr><h3>Help and Information</h3><p>Many resources to help you can be found here together with additional links to more resources, documentation, support and how to become involved in Joomla.</p>"
```
Comment (brianteeman) [inline administrator/language/en-GB/guidedtours.joomla_whatsnew_5_2_steps.ini]:
```suggestion
COM_GUIDEDTOURS_TOUR_WHATSNEW_5_2_STEP_0_DESCRIPTION="<h3>Joomla 5.2 Full Release Notes</h3><p><a title=\"Full release notes\" href=\"https://www.joomla.org/announcements.html\" target=\"_blank\" rel=\"noopener noreferrer\">View the full release notes on joomla.org</a></p><hr><h3>Help and Information</h3><p>Many resources to help you can be found here together with additional links to more resources, documentation, support and how to become involved in Joomla.</p>"
```

**Comment by brianteeman** [comment]
Thanks for confirming @obuisard I have suggested the correction now

File: administrator/language/en-GB/guidedtours.joomla_whatsnew_5_2_steps.ini (hunk 2 above)
Comment (obuisard) [reply administrator/language/en-GB/guidedtours.joomla_whatsnew_5_2_steps.ini]: Thanks Brian

File: administrator/language/en-GB/guidedtours.joomla_whatsnew_5_2.ini (hunk 1 above)
Comment (obuisard) [reply administrator/language/en-GB/guidedtours.joomla_whatsnew_5_2.ini]: Thanks Brian.

**Comment by tecpromotion** [comment]
RTC

**Comment by richard67** [comment]
> RTC

@tecpromotion ??? It doesn't have 2 successful human tests, or am I missing something? Approvals on GitHub just mean the PR is ok by review, but they are not equal to a test of the functionality by a human being.

**Comment by richard67** [comment]
Well I just see a review should be sufficient here. But still it needs "testers".

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 40955781467d89a934b9b43d16f2a93d6449daa0<br /><br />With a good diff tool I was able to compare the details of the changes in the language string, which the diff on GitHub doesn't show.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44075">issues.joomla.org/tracker/joomla-cms/44075</a>.</sub>

**Comment by obuisard** [comment]
I have tested this item :white_check_mark: successfully on 40955781467d89a934b9b43d16f2a93d6449daa0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44075">issues.joomla.org/tracker/joomla-cms/44075</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44075">issues.joomla.org/tracker/joomla-cms/44075</a>.</sub>

**Comment by Hackwar** [comment]
Thanks for your contribution @obuisard and @brianteeman

**Pull request merged**